hedera-node/src/test/java/com/hedera/services/context/primitives/StateViewTest.java

```diff
@@ -489,6 +494,26 @@ void getsTokenInfoMinusFreezeIfMissing() {
 		assertFalse(info.hasFreezeKey());
 	}
 
+	@Test
+	void getsTokenInfoMinusPauseIfMissing() {
+		// setup:
+		token.setPauseKey(MerkleToken.UNUSED_KEY);
+
+		// when:
+		var info = subject.infoForToken(tokenId).get();
+
+		// then:
+		assertEquals(tokenId, info.getTokenId());
+		assertEquals(token.symbol(), info.getSymbol());
+		assertEquals(token.name(), info.getName());
+		assertEquals(token.treasury().toGrpcAccountId(), info.getTreasury());
+		assertEquals(token.totalSupply(), info.getTotalSupply());
+		assertEquals(token.decimals(), info.getDecimals());
+		assertEquals(TOKEN_ADMIN_KT.asKey(), info.getAdminKey());
+		assertEquals(PauseNotApplicable, info.getPauseStatus());
+		assertFalse(info.hasPauseKey());
+	}
+
```

Comment:
Don't just copy-paste. These should be refactored along the way when a class / test suite is touched. The comments of `// setup: // when: // then:` do not add any value as they explain what you are doing (should be avoided as the code should tell) instead of why you are doing it this way. The `var info` should be `final var info`. These are among a few things we should do throughout our code to increase the code quality.